flexwrfoutput/openfiles.py

```diff
@@ -0,0 +1,77 @@
+"""
+Functions to handle raw netCDF output of FLEXPART-WRF.
+
+Meant to be used before the postprocessing with the accessor.
+"""
+from pathlib import Path
+from typing import Tuple, Union
+
+import xarray as xr
+
+
+def combine(flxout: xr.Dataset, header: xr.Dataset) -> xr.Dataset:
+    """Combines dimensions of flxout with header to have full information of output in\
+        one xarray.
+
+    Args:
+        flxout (xr.Dataset): Loaded flxout file.
+        header (xr.Dataset): Loader header file.
+
+    Returns:
+        xr.Dataset: Combined Dataset
+    """
+    combined = xr.merge([flxout, header.drop_dims("Time")])
+    return combined
+
+
+class AmbiguousPathError(Exception):
+    pass
+
+
+def get_output_paths(path: Union[str, Path]) -> Tuple[Path, Path]:
```

Comment:
Jetzt wo die Funktionen nicht mehr exposed sind würde ich sie mit underscores prefacen, damit klar ist, dass sie intern sind.

```suggestion
def _get_output_paths(path: Union[str, Path]) -> Tuple[Path, Path]:
```